src/natcap/invest/sdr/sdr.py

```diff
@@ -66,7 +66,7 @@
                 "but in case of its absence, there are methods and equations "
                 "to help generate a grid using climatic data.  The units are "
                 "MJ*mm/(ha*h*yr)."),
-            "name": "Rainfall Erosivity Index (R)"
+            "name": "rainfall erosivity index"
```

Comment:
After more consideration, I think it's better to leave out the equation parameters everywhere... it would be easy for them to get out-of-sync with the parameters in the user's guide. Better to stick with descriptive names. I don't think these are "standard" variable names like in the way that _t_ commonly stands for time.

Reply:
I agree, let's stick with descriptive names and leave out the equation parameters where possible.  I myself am not always clear on what's standard in the field or if we've just invented some 'standard' names and stuck with them internally.